### PR TITLE
Dockerfile.native - run with non-root user 1001

### PR DESCRIPTION
**Pull request by haraldatbmw**
The native docker-image should run with non-root user 1001.

### CODE REVIEW

**Review by gsmet — APPROVED** [review]
Thanks!

Makes sense to me but I prefer we have @cescoffier 's blessing before merging.

**Comment by gsmet** [comment]
@cescoffier can you check that one?

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
Thanks! That was a bug actually (never run as root)

**Pull request merged**
